assets/airports/lkpr.json

```diff
@@ -400,85 +1433,150 @@
             ]
         },
         {
-            "origin": "",
-            "destination": "lkpr",
-            "category": "arrival",
-            "route": "LOMKI091.LOMKI.LKPR",
-            "altitude": 12000,
-            "speed": 320,
-            "method": "random",
-            "rate": 8,
+            "origin": "LKPR",
+            "destination": "",
+            "category": "departure",
+            "route": "LKPR24.BALTU3A.BALTU.L984.OKG",
+            "altitude": "",
+            "speed": "",
+            "method" : "random",
+            "rate": 1,
             "airlines": [
-                ["TVS", 16],
-                ["CSA", 16],
-                ["EZY", 12],
-                ["RYR",8],
+                ["TVS", 20],
+                ["CSA", 20],
+                ["EZY", 10],
+                ["RYR",10],
                 ["AFR",4],
+                ["AUA",4],
                 ["AZA",4],
+                ["AFL",4],
                 ["BAW",4],
                 ["DLH",4],
                 ["SWR",4],
                 ["KLM",2],
                 ["KLC",2],
```

Comment:
Replace all "KLC" entries with "KLM/embraers", KLC only operates under KLM callsign.

Reply:
Replaced KLM and added fleets where that was possible. 

I was made aware to request review from @openscope/airport-reviewers by @Fechulo yesterday, but since I was unaware of that beforehand, github showed you, @indianbhaji and Nate as suggested reviewers and so I requested from you. Will request from reviewers next time, or should I re-request it @Marcel510 ?